src/sick_tim551_2050001.cpp

```diff
@@ -60,7 +63,7 @@ int main(int argc, char **argv)
   if (subscribe_datagram)
     s = new sick_tim::SickTimCommonMockup(parser);
   else if (useTCP)
-    s = new sick_tim::SickTimCommonTcp(hostname, port, parser);
+    s = new sick_tim::SickTimCommonTcp(hostname, port, timelimit, parser);
```

Comment:
`timelimit` should be an int, not a string (don't be fooled by `port`; that's a string because `tcp::resolver::query` wants a string there).